libs/blocks/modal/modal.css

```diff
@@ -1,5 +1,7 @@
 :root {
   --modal-focus-color: #109cde;
+  --spacing-xl: 56px;
```

Comment:
Shouldn't we scope this to the modal element? Otherwise we'd essentially overwrite https://github.com/adobecom/milo/blob/stage/libs/styles/styles.css#L49 on pages using modals. For viewports wider than `1200px`, the variable's value will match yours anyway - https://github.com/adobecom/milo/blob/stage/libs/styles/styles.css#L171.

So: do we need this value to be different? If so, let's scope it to `.dialog-modal`.

Reply:
>So: do we need this value to be different? If so, let's scope it to .dialog-modal.

No, I was just trying carry that variable over to this modal. I must have missed it being accessible already. I'll make the correction.

Reply:
Rather then redeclaring this spacing token,  can you just use `--spacing-xl-static` in styles.css?